.github/workflows/post-merge-jobs.yml

```diff
@@ -60,15 +60,30 @@ jobs:
                   global-json-file: "./global.json"
             - name: Restore .net tools
               run: dotnet tool restore
-            - name: Make build directory
-              run: mkdir ./build
+            - name: Make build and temp directories
+              run: |
+                  mkdir ./build
+                  mkdir ./temp
             - name: Build Jobs
               run: |
                 cd src/Aquifer.Jobs
                 dotnet build --configuration Release --output ../../build --runtime win-x64
                 cd ../..
             - name: Bundle migrations
               run: dotnet ef migrations bundle --startup-project src/Aquifer.Migrations --project src/Aquifer.Data --context AquiferDbContext --output build/Migrate --self-contained
+            - name: Download ffmpeg from Azure Blob Storage
+              uses: armanrahman22/azblob-download-action
+              with:
+                  storage-account-name: "aquiferstoragedev"
+                  container-name: "deployment-artifacts"
```

Comment:
I created this new container in `aquiferstoragedev` and granted the GitHub Actions user permission to read Blob Storage.